src/vpoller/proxy.py

```diff
@@ -131,6 +133,16 @@ def load_config(self):
             self.config
         )
 
+    def check_proxy(self):
```

Comment:
Can you please add a docstring for check_proxy() function, so that you are consistent with the other functions?